### PR TITLE
Documentation updates and minor bug fixes

### PR DESCRIPTION
**Pull request by joelhegg**
I've been taking a closer look at this code base and noticed some documentation issues. I also fixed some related minor bugs in `isRequestFromAssistant` and `buildLineItem`.

### CODE REVIEW

File: actions-sdk-app.js [hunk 1]
```diff
@@ -111,12 +111,12 @@ class ActionsSdkApp extends AssistantApp {
...
         reject(errorMsg);
       }
-      googleAuthClient.verifyIdToken(jwtToken, projectId, (err, idToken) => {
+      googleAuthClient.verifyIdToken(jwtToken, projectId, (err, login) => {
         if (err) {
           error('ID token verification Failed: ' + err);
           reject(err);
         } else {
-          resolve(idToken);
+          resolve(jwtToken);
```
Comment (smishra2) [inline actions-sdk-app.js]: This seems to be a breaking change, can you explain the reason for returning the jwtToken?

File: assistant-app.js [hunk 2]
```diff
@@ -1639,12 +1639,12 @@ class AssistantApp {
...
    * Constructs LineItem with chainable property setters.
    *
-   * @param {string} name Name of the line item.
    * @param {string} id Unique identifier for the item.
+   * @param {string} name Name of the line item.
    * @return {LineItem} Constructed LineItem.
    */
-  buildLineItem (name, id) {
-    return new LineItem(name, id);
+  buildLineItem (id, name) {
```
Comment (smishra2) [inline assistant-app.js]: This also seems to be a breaking change. Could be a revision for the next major version release

File: actions-sdk-app.js (hunk 1 above)
Comment (joelhegg) [reply actions-sdk-app.js]: The issue here is the implementation didn't match the documentation. From the documentation:
> Promise resolving with ID token if request is from a valid source

Here's what the call in question looked like:
```
  googleAuthClient.verifyIdToken(jwtToken, projectId, (err, idToken) => {
```

Here's what the actual signature of that method looks like:
```
  public verifyIdToken(
    idToken: string, audience: string\|string[],
    callback: (err: Error, login?: LoginTicket) => void) {
```

So there are two problems here.
1. The second callback parameter is a `LoginTicket` not an ID token. Returning that conflicts with the documentation.
1. If you want to return what googleAuthClient considers to be the ID token, then it's the first parameter, which in this case is `jwtToken`.

File: assistant-app.js (hunk 2 above)
Comment (joelhegg) [reply assistant-app.js]: Here's the constructor for `LineItem`:
```
onstructor (lineItemId, name) {

**Comment by joelhegg** [comment]
@smishra2 See my responses. Let me know if you'd still like me to revert the code changes and create separate issues.

**Comment by joelhegg** [comment]
Created issues #56 and #57 to track the code issues.

**Comment by joelhegg** [comment]
I'm going to close this since the related issues took care of the bugs.

**Comment by Canain** [comment]
Hi, the code changes we made were just for the two issues you filed. Do you still want to contribute the `transactions.js` JSDoc changes?

**Comment by joelhegg** [comment]
@Canain I ran across a number of other jsdoc issues. I just submitted them in #63.

**Pull request closed without merging**